dataminer/Administrator_guide/Databases/Indexing_database/OpenSearch_Database/Installing_OpenSearch_database.md

```diff
@@ -474,3 +477,35 @@ To monitor expiry dates, we recommend using the **Skyline SSL Certificate Monito
 1. [Create an alarm template](xref:Creating_an_alarm_template) to monitor the parameter *Remaining Days* of the protocol, and [assign it to the element](xref:Assigning_an_alarm_template).
 
    ![Alarm template configuration](~/dataminer/images/Skyline_SSL_Certificate_Monitor_thresholds.png)
+
+## Renewing the TLS certificates for OpenSearch
+
+To renew your self-signed TLS certificates for an existing OpenSearch database, you will have to replace the existing certificates with newly signed ones. The expired certificates can no longer be used. To generate new certificates, we recommend using the *Generate-TLS-Certificates* script.
+
+1. Generate the certificates using the [Generate-TLS-Certificates](https://github.com/SkylineCommunications/generate-tls-certificates) script on GitHub.
+
+   This script will generate a .p12 file for every node and a .pem admin cert and key file that is used for security configuration settings. The GitHub script will provide you with the keystore and truststore password.
+
+   A rootCA.crt can also be generated by the GitHub script; however, we recommend using the original *rootCA.crt* (the same one used on the previous renewal/certification). If this is not possible, the new *rootCA.crt* will have to be applied on all DataMiner and OpenSearch nodes.
```

Comment:
If they use a new rootCA.crt, at which point should they apply this on the DataMiner nodes? Since we include the info about this right here, this gives the impression that users should do this at this point in the procedure, so if this isn't the case, we'll need to clarify this.